### PR TITLE
mds: Reorganize class members in MDSCacheObject header

### PR DESCRIPTION
**Pull request by varshar16**
Fixes: https://tracker.ceph.com/issues/42329
Signed-off-by: Varsha Rao <varao@redhat.com>

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

**Comment by varshar16** [comment]
jenkins retest this please

**Comment by varshar16** [comment]
jenkins retest this please

File: src/mds/MDSCacheObject.h [hunk 1]
```diff
@@ -70,55 +72,29 @@ class MDSCacheObject {
   static const int PIN_CLIENTLEASE = 1009;
   static const int PIN_DISCOVERBASE = 1010;
```
Comment (joscollin) [inline src/mds/MDSCacheObject.h]: Treat these as data members rather than constants to have a better arrangement.
Otherwise, looks good to me.

Comment (varshar16) [reply src/mds/MDSCacheObject.h]: Since we are following the same order in other header files. Let's not change it.

**Pull request merged**